README.md

```diff
@@ -223,6 +225,36 @@ app.use(function (err, req, res, next) {
 })
 ```
 
+### Verifying a token
+
+If you need to pass the token in a query string rather than an
+HTTP header, you can use the `req.verifyToken(token)` function
+provided by this middleware. This is useful when you want to
+use this library with an oauth callback.
+
+```js
+var csrf = require('csurf')
+var express = require('express')
+
+var app = express()
+app.use(csrf())
+
+app.get('/oauth_callback', function(req, res) {
+  try {
+    req.verifyToken(req.query.state); // throws
```

Comment:
> This example does not provide a use-case, though.

I'm not sure what you're looking for. The example demonstrates how to use the more explicit token verification function as an alternative to the `value` option. The situation where this is useful is a bit difficult to demonstrate because I'd have to concoct some complicated example where the `value` option isn't sufficient (in my case, for example, deep within _one_ express route where I'd rather be explicit as to how that route is different). If you want me to give an example like that, it'd fill the example with so much unnecessary garbage that nobody would really understand what the function does or how to use it.

This example, on the other hand, gives a _great_ example of what the function does and how to use it. So I'm not really sure what you're suggesting.